sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -325,19 +286,36 @@ def open_db_connections(self, db_key, db_name=None, is_default=True, key_prefix=
         except Exception as e:
             error_message = self.test_network_connectivity()
             tcp_connection_status = error_message if error_message else "OK"
-            message = "Unable to connect to SQL Server (host={} database={}). TCP-connection({}). Exception: {}".format(
-                host, database, tcp_connection_status, _format_connection_exception(e)
-            )
+            exception_msg, conn_warn_msg = format_connection_exception(e, driver)
+            if tcp_connection_status != "OK" and conn_warn_msg is ConnectionErrorCode.unknown:
```

Comment:
`:meh:` Another old reference to warning? 
```suggestion
            if tcp_connection_status != "OK" and conn_err_msg is ConnectionErrorCode.unknown:
```